archetypes/archetype-app-quickstart/README.md

```diff
@@ -43,12 +43,13 @@ mvn archetype:generate \
 ### Parameters
       
 Parameter Name | Default Value | Description
----|---|---
+---|-----|---
 `service` (required) | n/a | Specifies the service client to be used in the application, eg: s3, dynamodb. Only one service should be provided. You can find available services [here][java-sdk-v2-services]. 
 `groupId`(required) | n/a | Specifies the group ID of the project
 `artifactId`(required) | n/a | Specifies the artifact ID of the project
 `nativeImage`(required)  | n/a | Specifies whether GraalVM Native Image configuration should be included
 `httpClient`(required) | n/a | Specifies the http client to be used by the SDK client. Available options are `url-connection-client` (sync), `apache-client` (sync), `netty-nio-client` (async). See [http clients][sdk-http-clients]
+`ssooidc` (required) | n/a | Specify if the sso and ssoidc dependency should be included in the project. Include them to enable the use of [IAM Identity center](https://aws.amazon.com/iam/identity-center/) for credentials.
```

Comment:
Should we mention the sso and ssoidc or mention someting which is easier for user to undersatnd
"specifc SSO Credetial Provider or SSO TokenProvider related dependencies "

I guess this request is very specifc to SSO Token Provider.
 Note : If we User wants only SSO Credential provider than we donot need sso-oidc dependency , we can clarify this with the requester of this feature.

On other note we can make it generic so that we can add future dependencies related to Credentials providers like STS for example.

Example something like "Press  following option to auto add the depencires related to Credentials Providers 1 for SSO Credentials  2.  SSO Token Providers  3. STS Credentials providers"

when 1 we just add SSO 
when 2 we add SSO + SSO-OIDC
when 3 we add STS (for future not required in this PR)

Reply:
How about `-DcredentialProvider=(default|identity-center)` for now, with `identity-center` adding both `sso` and `ssooidc` dependency

Reply:
In the future we'll need to widen the terminology to reference identity providers, but for now this should be fine. I think the refactoring of the SDK with regards to new modules will eventually change the setup scripts quite a lot.